backend/utils/DatabaseConnector.go

```diff
@@ -0,0 +1,39 @@
+package utils
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/fourtf/studyhub/models"
+	"github.com/jinzhu/gorm"
+
+	//Needed to use the postgres driver
+	_ "github.com/jinzhu/gorm/dialects/postgres"
+)
+
+//ConnectToDB establishes a connection to the database and handles errors if the database is not available
+func ConnectToDB() *gorm.DB {
+	LoadEnvironmentVariables()
+
+	username := os.Getenv("databaseUser")
+	password := os.Getenv("databasePassword")
+	databaseName := os.Getenv("databaseName")
+	databaseType := os.Getenv("databaseType")
+
+	connectionString := fmt.Sprintf("user=%s dbname=%s password=%s sslmode=disable", username, databaseName, password)
+
+	db, err := gorm.Open(databaseType, connectionString)
+
+	if err != nil {
+		log.Println("error", err)
+		panic(err)
```

Comment:
Returning the error and handling it in main would be preferred. Also use os.Exit instead of panic.

Excerpt from StackOverflow:
1.  Use panic when something really weird happens, e.g. programming logic error.
2. Use os.Exit when you want an immediate exit, with specified exit code.